sdk/src/user.ts

```diff
@@ -2937,12 +2937,15 @@ export class User {
 			withdrawLimit = BN.max(withdrawLimit, userDepositAmount);
 		}
 
-		const amountWithdrawable = freeCollateral
-			.mul(MARGIN_PRECISION)
-			.div(new BN(spotMarket.initialAssetWeight))
-			.mul(PRICE_PRECISION)
-			.div(oracleData.price)
-			.mul(precisionIncrease);
+		const amountWithdrawable =
+			spotMarket.initialAssetWeight === 0
```

Comment:
i think we want to load the `initialAssetWeight` from the sdk function rather than just reading the account value